workloads/baseline-performance/README.md

```diff
@@ -0,0 +1,18 @@
+ # Baseline Cluster Performance benchmark
+
+The baseline performance benchmark is used to measure the performance metrics of the cluster when there are no resources loaded on the cluster. Helps in understanding the baseline performance of the control plane components, nodes, etcd, kubelet etc of Openshift. The baseline workload will sleep for ${WATCH_TIME} minutes and call kube-burner indexer to collect the metrics without loading the cluster.
+
+
+
+## Common variables
+
+These environment variables can be customized 
+
+| Variable         | Description                         | Default |
+|------------------|-------------------------------------|---------|
+| **KUBE_BURNER_RELEASE_URL** | kube-burner tarball release location | `https://github.com/cloud-bulldozer/kube-burner/releases/download/v0.9.1/kube-burner-0.9.1-Linux-x86_64.tar.gz` |
+| **WATCH_TIME**              | Sleep duration for which metrics will be collected in minutes| 15 |
```

Comment:
We discussed bumping this to 30

Reply:
done , had to run off immediately after the call 😂